src/moma.cpp

```diff
@@ -56,6 +58,9 @@ MoMA::MoMA(const arma::mat &i_X,  // Pass X_ as a reference to avoid copy
                i_X.n_cols)
 // const reference must be passed to initializer list
 {
+    ds         = i_ds;
+    X_original = i_X;
```

Comment:
Any reason these are non-const?

Reply:
I thought about that too. A const member must be initialized in the initiation list. And there are two initialization method in the C++ class MoMA, one relying on the other:

https://github.com/DataSlingers/MoMA/pull/54/files#r316588764

And C++ does not allow both of them to present at the initialization list, i.e., can't have

`MoMA(...): MoMA(....), X_original(i_X) {}`

Reply:
#61